packages/stakers/src/set/setMevBoost.ts

```diff
@@ -22,7 +23,7 @@ export async function setMevBoost<T extends Network>({
   currentMevBoostPkg,
 }: {
   dappnodeInstaller: DappnodeInstaller;
-  mevBoost: T extends "mainnet" ? MevBoostMainnet : MevBoostPrater;
+  mevBoost: T extends "mainnet" ? MevBoostMainnet : MevBoostPrater | MevBoostHolesky;
```

Comment:
This looks incorrect, it would be:
`T extends "mainnet" ? MevBoostMainnet : T extends"prater" ? MevBoostPrater : MevBoostHolesky;`
or something similar